external/quarkus_quickstarts/README.md

```diff
@@ -0,0 +1,69 @@
+# External (Third Party Container) Tests
```

Comment:
Please change heading from
External (Third Party Container) Tests 
to 
External Quarkus Quickstart Tests